src/services/sizes.js

```diff
@@ -0,0 +1,35 @@
+const { Sizes } = require('../models');
+
+class SizesService {
+  async createSize (req, res) {
+    const doesExist = await Sizes.exists(req.body);
+    if (doesExist) {
+      return res.boom.badRequest('Size already exists.');
+    }
+    const size = await new Sizes(req.body).save();
+
+    return res.json(size);
```

Comment:
Return true or false in create, update and delete methods. Let's follow the same pattern